libs/ui-toolkit/datepicker/src/lib/datepicker-toggle.component.ts

```diff
@@ -50,4 +57,34 @@ export class DatepickerToggleComponent<D> extends MatDatepickerToggle<D> {
   @HostListener('focus') public setFocus() {
     this._button.focus();
   }
+
+  public ngOnInit() {
```

Comment:
Does MatDatepickerToggle have a ngOnInit method? Is it necessary to call it here as well?

Reply:
Don't have it